test/simulator/timestamp/src/include/session_simulator.h

```diff
@@ -36,6 +38,11 @@ class session_simulator {
     void begin_transaction();
     void rollback_transaction();
     void commit_transaction();
+    int timestamp_transaction_uint(const std::string &, const uint64_t &);
+    void set_commit_timestamp(const uint64_t &);
```

Comment:
As discussed, we can simply pass the argument as non-const and by value. By definition, a setter should not do anything than use the value and assign it to a private member.

Reply:
What about in `timestamp_transaction_uint`?

Reply:
`session_simulator::timestamp_transaction_uint(const std::string &ts_type, uint64_t ts)`

Reply:
Fixed.